CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 
 - CCF is now built with Clang 10. It is recommended that C++ applications upgrade to Clang 10 as well.
+- Internal `/gov/jwt_keys/refresh` endpoint has been moved to `/node/jwt_keys/refresh`.
```

Comment:
This endpoint isn't listed in the OpenAPI spec yet it shows up in the metrics for a given frontend so I've listed it here just in case.